assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/empty-mini-cart-contents-block/edit.tsx

```diff
@@ -44,7 +44,7 @@ export const Edit = ( { clientId }: { clientId: string } ): JSX.Element => {
 			}
 		>
 			<InnerBlocks
-				allowedBlocks={ allowedBlocks }
```

Comment:
We will want to prevent some blocks to be added here, but we can do it in a follow-up issue/PR once this one is merged.